speakers.js

```diff
@@ -0,0 +1,88 @@
+const speakers = [{
```

Comment:
We need to follow the design for this project but can only change the content. As the previous reviewer mentioned please stick to the design guideline provided and make the speaker's image and background square instead of a circle. 


<img width="1059" alt="Screenshot 2023-03-17 at 9 12 38 PM" src="https://user-images.githubusercontent.com/25887253/225985416-fe424db4-c881-432e-bea1-f98e8ef46e76.png">
 
<img width="678" alt="Screenshot 2023-03-17 at 9 05 34 PM" src="https://user-images.githubusercontent.com/25887253/225985348-c06fede8-04d4-4582-bf45-eb4ef9b0a84a.png">